test_scripts/Smoke/API/015_Show_PositiveCase.lua

```diff
@@ -0,0 +1,134 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: Show
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [Show] SUCCESS: getting SUCCESS:UI.Show()
+--
+-- Description:
+-- Mobile application sends valid Show request and gets UI.Show "SUCCESS" response from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests Show with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if UI interface is available on HMI
+-- SDL checks if Show is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the UI part of request with allowed parameters to HMI
+-- SDL receives UI part of response from HMI with "SUCCESS" result code
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local putFileParams = {
+  requestParams = {
+    syncFileName = 'icon.png',
+    fileType = "GRAPHIC_PNG",
+    persistentFile = false,
+    systemFile = false
+  },
+  filePath = "files/icon.png"
+}
+
+local requestParams = {
+  mainField1 = "a",
+  mainField2 = "a",
+  mainField3 = "a",
+  mainField4 = "a",
+  statusBar = "a",
+  mediaClock = "a",
+  mediaTrack = "a",
+  alignment = "CENTERED",
+  graphic = {
+    imageType = "DYNAMIC",
+    value = "icon.png"
+  },
+  secondaryGraphic = {
+    imageType = "DYNAMIC",
+    value = "icon.png"
+  },
+}
+
+local responseUiParams = {
+  showStrings = {
+    {
+      fieldName = "mainField1",
+      fieldText = requestParams.mainField1
+    },
+    {
+      fieldName = "mainField2",
+      fieldText = requestParams.mainField2
+    },
+    {
+      fieldName = "mainField3",
+      fieldText = requestParams.mainField3
+    },
+    {
+      fieldName = "mainField4",
+      fieldText = requestParams.mainField4
+    },
+    {
+      fieldName = "mediaClock",
+      fieldText = requestParams.mediaClock
+    },
+    {
+      fieldName = "mediaTrack",
+      fieldText = requestParams.mediaTrack
+    },
+    {
+      fieldName = "statusBar",
+      fieldText = requestParams.statusBar
+    }
+  },
+  alignment = requestParams.alignment,
+  graphic = {
+    imageType = requestParams.graphic.imageType,
+    value = commonSmoke.getPathToFileInStorage(requestParams.graphic.value)
```

Comment:
@GetmanetsIrina ```getPathToFileInStorage()``` needs to be corrected.
It uses ```commonSmoke.getPathToSDL()```  function, which reads the data from config.pathToSDL
But this parameter can be with (or without) slash at the end
So it's better to use ```GetPathToSDL()``` function from ```commonPreconditions``` module which handles these cases correctly

Reply:
@dboltovskyi , Please find update in 76c706f6581df43f9ee00c457e2a7fad7535eeb9